src/unknown_008C40.h

```diff
@@ -2,14 +2,29 @@
 #define _UNKNOWN_008C40_H_
 
 #include "types.h"
+
+f32 sqrtf(f32);
```

Comment:
Is this necessary?

Reply:
To silence warnings it needed something. I've settled on including math.h for now.